src/components/sections/Hero.section.tsx

```diff
@@ -1,59 +1,68 @@
-import React, { useState, useEffect, useCallback } from 'react';
+import React, { useState, useEffect, useCallback } from "react";
 import {
-  HeroIllustration,
-  LinkedInIcon,
-  TwitterIcon,
-  InstagramIcon,
-  DiscordIcon
-} from '@assets';
+	HeroIllustration,
+	LinkedInIcon,
+	TwitterIcon,
+	InstagramIcon,
+	DiscordIcon,
```

Comment:
This is a crazy amount of spacing. Is this 4 space for a tab?

Reply:
I'm using prettier for formatting, looks like the file was set to have indentation to use spaces instead of tabs